introduction.md

```diff
@@ -1,3 +1,7 @@
 I'm Alejandro Greppi, doing this Github course on CodeSchool.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?